plugins/content-length.rst

```diff
@@ -1,4 +1,35 @@
 Content-Length Plugin
 =====================
 
-TODO
+The `ContentLengthPlugin` sets the correct `Content-Length` header value based on the size of the body stream of the
+request. This helps HTTP servers to handle the request.
+
+If the size of the stream can not be determined, the plugin sets the Encoding header to `chunked`, as defined in
+:rfc:`7230#section-4.1`::
```

Comment:
this makes me expect the code example is about junked, which is wrong. maybe: move this line up after "... the body stream of the request." and then have the "helps HTTP server" as prompt for the code? "Specifying the content length in the header helps HTTP servers to handle the request more efficiently::"